projects/planner/src/app/learn/learn.component.html

```diff
@@ -7,8 +7,8 @@
             <h2 class="accordion-header">
               <button
                 class="accordion-button"
-                [class.collapsed]="activeTopic !== topic.topic"
-                (click)="toggleTopic(topic.topic)"
+                [class.collapsed]="selectedTopic.topic !== topic.topic"
```

Comment:
see other comments => simplify also the UI, which no longer needs to compare by name.